libs/tracing/src/main/kotlin/net/corda/tracing/TracingService.kt

```diff
@@ -6,7 +6,32 @@ import java.util.concurrent.ExecutorService
 
 interface TracingService : AutoCloseable {
 
-    fun addTraceHeaders(headers: List<Pair<String, String>>): List<Pair<String, String>>
+    // The method takes the current headers and joins them with the trace headers
+    // A trace context provides the trace headers. Normally, the activate context is used, but it can be overridden
```

Comment:
```suggestion
    // A trace context provides the trace headers. Normally, the active context is used, but the headers can be overridden
```

Similar sort of change should be applied to all the methods below. Also, it would  probably makes sense to change the format to a KDoc rather than a comment.